python_syntax.md

```diff
@@ -355,3 +355,323 @@ def foo():
 ### 7. Imports
 
 Sort imports using `isort`.
+
+### 8. Docstrings and Comments
+
+Docstring conventions used at Makimo are mostly based on the three following
+documents (which we recommend to read):
+
+1. [PEP 257 - Docstring Conventions](https://www.python.org/dev/peps/pep-0257/)
+   (Accepted, necessary rules)
+2. [PEP 224 - Attribute Docstrings](https://www.python.org/dev/peps/pep-0224/)
+   (Officially rejected, but used as a convention anyway)
+3. [Google Python Style Guide, 3.8 Comments ands Docstrings](
+https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings)
+4. [Example Google Style Python Docstrings](
+https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)
+
+Regarding usage of comments and docstrings, the basic rule is this:
```

Comment:
The rule of thumb is to use docstrings whenever documenting public interface. Comments should be left to such cases as laying out a structure and inner workings of the code with the expectation that they help the programmer modifying this particular file or module.

How do you feel on this wording?